src/Razor/test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/ModelExpressionTagHelper_DesignTime.codegen.cs

```diff
@@ -13,6 +13,11 @@ namespace AspNetCore
     using Microsoft.AspNetCore.Mvc.ViewFeatures;
     public class TestFiles_IntegrationTests_CodeGenerationIntegrationTest_ModelExpressionTagHelper : global::Microsoft.AspNetCore.Mvc.Razor.RazorPage<DateTime>
     {
+        #line hidden
+        #pragma warning disable 0649
```

Comment:
This suppresses the error `Field 'XYZ' is never assigned to`. Alternatively I could have rendered out the normal runtime execution context; however, that would mess with VS formatting because all children would be in a new block scope. I could also just set the executioncontext here to `null` but decided against that in an effort to not further the gap of nullability compliant codegen.

Reply:
I super agree that this is the right thing to do.